Sources/Bow/Instances/NumberInstances.swift

```diff
@@ -14,6 +14,17 @@ extension Int: Monoid {
     }
 }
 
+/// Instance of `Semiring` for `Int`. Uses multiplication as multiplication of elements and one as one element.
```

Comment:
```suggestion
/// Instance of `Semiring` for `Int`. Uses product (`*`) as multiplication of elements and `1` as one element.
```

Reply:
Can you change the rest of the comments accordingly?

Reply:
@traviscibot updated :) and yes I have run all the tests.